cost/flexera/cco/cheaper_regions/cheaper_regions.pt

```diff
@@ -329,11 +368,11 @@ end
 # Policy
 ###############################################################################
 
-policy "policy_cheaper_regions" do
+policy "pol_cheaper_regions" do
   validate_each $ds_new_bc_cost_name do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows with cheaper data"
-    check eq(0,1)
-    escalate $email
+    check eq(0, 1)
```

Comment:
The check statement should only raise an incident if something worth reporting was found. A check statement like this will *always* raise an incident, even if no cheaper regions have been found.